xmppserver/src/main/java/org/jivesoftware/openfire/http/HttpBindManager.java

```diff
@@ -94,7 +93,7 @@ public final class HttpBindManager implements CertificateEventListener {
     public static final SystemProperty<Integer> HTTP_BIND_SECURE_PORT = SystemProperty.Builder.ofType(Integer.class)
         .setKey("httpbind.port.secure")
         .setDynamic(true)
-        .setDefaultValue(7443)
+        .setDefaultValue(7444)
```

Comment:
Ah, this is an accidental change that is removed in a later commit. Please squash that commit with this commit (or edit this commit, and drop the other commit).